cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -284,6 +284,8 @@ def parse_harvard_opinions(reporter, volume, make_searchable):
             .replace("Docket Nos.", "")
             .strip()
         )
+        if len(docket_string) > 5000:
+            docket_string = "See below for full docket number"
```

Comment:
I'd be more explicit here. "See corrections field for full docket number." "Below" might become above aside right left, etc.